configMaps/config_map.yml.mustache

```diff
@@ -0,0 +1,153 @@
+                </com.cloudbees.jenkins.plugins.bitbucket.OriginPullRequestDiscoveryTrait>
+                <com.cloudbees.jenkins.plugins.bitbucket.ForkPullRequestDiscoveryTrait>
+                  <strategyId>1</strategyId>
+                  <trust class="com.cloudbees.jenkins.plugins.bitbucket.ForkPullRequestDiscoveryTrait$TrustTeamForks"/>
+                </com.cloudbees.jenkins.plugins.bitbucket.ForkPullRequestDiscoveryTrait>
+                <jenkins.plugins.git.traits.SubmoduleOptionTrait plugin="git@4.0.0">
+                  <extension class="hudson.plugins.git.extensions.impl.SubmoduleOption">
+                    <disableSubmodules>false</disableSubmodules>
+                    <recursiveSubmodules>true</recursiveSubmodules>
+                    <trackingSubmodules>false</trackingSubmodules>
+                    <reference></reference>
+                    <parentCredentials>true</parentCredentials>
+                    <shallow>false</shallow>
+                  </extension>
+                </jenkins.plugins.git.traits.SubmoduleOptionTrait>
+              </traits>
+            </source>
+            <strategy class="jenkins.branch.DefaultBranchPropertyStrategy">
+              <properties class="empty-list"/>
+            </strategy>
+          </jenkins.branch.BranchSource>
+        </data>
+        <owner class="org.jenkinsci.plugins.workflow.multibranch.WorkflowMultiBranchProject" reference="../.."/>
+      </sources>
+      <factory class="org.jenkinsci.plugins.workflow.multibranch.WorkflowBranchProjectFactory">
+        <owner class="org.jenkinsci.plugins.workflow.multibranch.WorkflowMultiBranchProject" reference="../.."/>
+        <scriptPath>.agile-cicd/build.Jenkinsfile</scriptPath>
+      </factory>
+    </org.jenkinsci.plugins.workflow.multibranch.WorkflowMultiBranchProject>
+  build-state.xml: |-
+    <?xml version='1.1' encoding='UTF-8'?>
+    <jenkins.branch.MultiBranchProject_-State plugin="branch-api@2.5.4">
+      <sourceActions>
+        <entry>
+          <string>9c19f900-3f7b-4f13-818d-7e0088101389</string>
+          <list>
+            <com.cloudbees.jenkins.plugins.bitbucket.BitbucketRepoMetadataAction plugin="cloudbees-bitbucket-branch-source@2.6.0">
+              <scm>git</scm>
+            </com.cloudbees.jenkins.plugins.bitbucket.BitbucketRepoMetadataAction>
+            <com.cloudbees.jenkins.plugins.bitbucket.BitbucketDefaultBranch plugin="cloudbees-bitbucket-branch-source@2.6.0">
+              <repoOwner>CSC</repoOwner>
+              <repository>${REPO_NAME}</repository>
+              <defaultBranch>develop</defaultBranch>
+            </com.cloudbees.jenkins.plugins.bitbucket.BitbucketDefaultBranch>
+            <com.cloudbees.jenkins.plugins.bitbucket.BitbucketLink plugin="cloudbees-bitbucket-branch-source@2.6.0">
+              <iconClassName>icon-bitbucket-repo</iconClassName>
+              <url>https://stash.agiledigital.com.au/projects/CSC/repos/${REPO_NAME}</url>

```

Comment:
Please don't hardcode this.

Reply:
Removed hardcoding. Github/bitbucket specifics are now supplied by users in jenkins